config/src/main/java/org/springframework/security/config/annotation/web/configurers/ExceptionHandlingConfigurer.java

```diff
@@ -127,6 +132,18 @@ public ExceptionHandlingConfigurer<H> defaultAccessDeniedHandlerFor(AccessDenied
 		return this;
 	}
 
+	public ExceptionHandlingConfigurer<H> defaultAuthenticationEntryPointFor(AuthenticationEntryPoint entryPoint,
```

Comment:
This name is still a bit confusing to me because it is configuring the access denied handler and does not clarify what the String type is. Perhaps something like `defaultDeniedHandlerForMissingAuthority`